src/base/rss/rssfeed.cpp

```diff
@@ -399,6 +399,32 @@ void Feed::downloadArticleTorrentIfMatching(const ArticlePtr &article)
     BitTorrent::Session::instance()->addTorrent(torrentUrl, params);
 }
 
+// find all occurrences of <placeholder> in original string, and replace them with value of "placeholder" from article
+// example: savePathTpl = "c:\downloads\<attr_name> - <description>\something\<title>\<not_found_attr>"
+//          article = <item>
+//                      <title>some title</title>
+//                      <link>some link</link>
+//                      <guid isPermaLink="false">some-guid</guid>
+//                      <pubDate>Thu, 02 Jun 2016 03:50:04 +0000</pubDate>
+//                      <description>some description</description>
+//                      <attr_name>some attr value</attr_name>
+//                    </item>
+//          result = "c:\downloads\some attr value - some description\something\some title\not_found_attr"
+QString Feed::generateSavePath(const QString &savePathTpl, const ArticlePtr &article)
+{
+    QRegExp rx("<(.+)>");
```

Comment:
All seem like good options. The exact placeholder syntax can be easily changed, of course.
I just picked something simple that works. I haven't thought about all the pros and cons of each and every option, and I wouldn't mind it being changed into anything that works well.

The addition of a default value is of course another good idea. That can be added as another feature.